aws/cloudformation/components/database.yml.erb

```
@@ -53,7 +53,7 @@
       Description: !Sub "Database credentials for ${AWS::StackName} writer user."
 <% if database -%>
       GenerateSecretString:
-        SecretStringTemplate: !Sub '{"username": "application-writer"}'
+        SecretStringTemplate: !Sub '{"username": "writer"}'
```

```suggestion
        SecretStringTemplate: '{"username": "writer"}'
```

`!Sub` is not necessary, as we're not actually doing any string interpolation. FWIW the single quotes _are_ necessary, otherwise it will get parsed into the template.

---

Fixed!